apps/auth/.env.prd

```diff
@@ -0,0 +1,25 @@
+EDUID_CLIENT_ID=uzh_klicker_auth_prd
+EDUID_WELL_KNOWN=https://login.eduid.ch/.well-known/openid-configuration
+NEXT_PUBLIC_EDUID_ID=eduid
+
+APP_ORIGIN_API=https://backend-sls.klicker.prd.df-app.ch
```

Comment:
this will all have to be changed to the real values (*.klicker.uzh.ch) for a real production deploy. we can leave it on these values until the final test though